tests/CMakeLists.txt

```diff
@@ -63,7 +61,7 @@ ELSE()
 
   IF("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
     MESSAGE(FATAL_ERROR "The testsuite cannot be configured in-source. "
-      "Please create a separate build directory"
+      "Please create a separate build directory!"
```

Comment:
Yes, it works correctly. It is CMake script and not bash - there is no shell invocation involved at this point.